app/main.py

```diff
@@ -81,22 +81,44 @@ def display_page(pathname):
 
 @app.callback(Output('alerts_info', 'children'),
               [Input('geojson_alerts', 'hover_feature'),
-              Input('markers', 'n_clicks'),
               Input('markers', 'hover_feature')])
```

Comment:
just a question here: I am not sure I understand where the 'hover_feature' come from for 'geojson_alerts' and 'markers', I have been searching in the code for the past 20 minutes and do not find it. Is it inherent to dl.Geojson layers and thus automatically generated? Thanks for the heads up :)

Reply:
Just a remark, should we let this whole section inside its own branch while it is still in development (we do not have the photos or real videos) and merge it when it's complete ?

Reply:
> Apart from that, I am not chocked by the video function being placed inside homepage.py. It could also be in utils.py I guess?

Yes of course. But I struggled to import functions from `homepage.py` into the other files (don't know why). So I prefered to stay in this file.

> Is it inherent to dl.Geojson layers and thus automatically generated?

Yes. It's part of the ~10 available properties of this components. ;)

> Just a remark, should we let this whole section inside its own branch while it is still in development (we do not have the photos or real videos) and merge it when it's complete ?

I'm not very keen of this solution because if we take too much delay to bring our first video, it will take some time to deal with conflict between the branch and the master that might have changed a lot.
I can maybe do like Sebastien for the graphs and disable it until we have proper data?

@Akilditu @pechouc  what do you think?

Reply:
Having an experiment branch could be an option here, let's discuss this tomorrow at our weekly meeting 👍